packages/portal/frontend/html/admin.html

```diff
@@ -165,21 +165,6 @@
         </ons-page>
     </template>
 
-    <template id="classlistDialog.html">
-        <ons-dialog id="adminClasslistConfirmHeader">
-            <ons-list-header style="text-align: center; font-weight: 700">
-            </ons-list-header>
-            <ons-scroller>
-                <ons-list>
```

Comment:
I think we can remove the CSS for this code too, as I do not see any JS references for #adminClasslistConfirmHeader.

```
#adminClasslistConfirmHeader ons-list {
    overflow-y: auto;
    height: 80vh;
}

#adminClasslistConfirmHeader ons-button {
    text-align: center;
    width: 100%;
}
```